src/clr-angular/forms/datepicker/providers/datepicker-view.service.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {isPlatformBrowser} from "@angular/common";
+import {ElementRef, Inject, Injectable, NgZone, PLATFORM_ID} from "@angular/core";
+import {first} from "rxjs/operator/first";
+
+/**
+ * This service focuses the day that is focusable in the calendar.
+ */
+@Injectable()
+export class DatepickerViewService {
```

Comment:
`DatepickerFocusService`? It only does one thing now.

Reply:
renamed to `DatepickerFocusService`